fdbclient/NativeAPI.actor.cpp

```diff
@@ -2784,6 +2788,11 @@ Future<Void> Transaction::commitMutations() {
 		if(options.firstInBatch) {
 			tr.flags = tr.flags | CommitTransactionRequest::FLAG_FIRST_IN_BATCH;
 		}
+		if(options.reportConflictingKeys) {
+			// TODO : Is it better to keep it as a flag?
+			tr.flags = tr.flags | CommitTransactionRequest::FLAG_REPORT_CONFLICTING_KEYS;
+			tr.reportConflictingKeys();
```

Comment:
There seems to be two (equivalent?) ways of indicating that we want to report conflict keys in the commit request. Let's just pick one.

Reply:
Thanks. Cleaned it.